bookkeeper-server/pom.xml

```diff
@@ -84,6 +84,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
In general I would prefer not to have to import transitive dependencies from low level libraries such as BookKeeper. Client code which uses BookKeeper will have to deal with a new dependency. For all my projects, which in turn are "libraries" this will add a new transitive depedency. As far as this library is only used for a simple "Pair" class I think it would be better to use a JDK class or a BookKeeper internal class.